### DISCUSSION TITLE
ways to update/change the data of existing chart?

### DISCUSSION THREAD

**Discussion by retsyo** [XusinboyBekchanov/MyFbFramework#39] [Q&A] [answered]
2 cases
1. only data is updated for existing, for example, bar chart
2. both axis and data need to be re-assigned, so that the chart is a totally new one

**Comment by XusinboyBekchanov** [comment, marked as answer]
1. You can change DoubleList values.
2. You can clear with command: Chart1.Clear. Then add items, axis items, series.
